content/posts/kubernetes/create-container-error.md

```diff
@@ -61,7 +61,9 @@ Warning  Failed                 8s (x3 over 22s)  kubelet, nginx-7ef9efa7cd-qasd
 ```
 
 then starting command might not be available on the image. Correct the container start command - or the image's contents - accordingly. See the [CrashLoopBackOff runbook]({{< relref "crashloopbackoff.md#solution-c" >}}) solution C.
+<!-- can this be removed? 
```

Comment:
What's the purpose of this link being displayed here? Also not properly formatted.